<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>GameWindow</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="widget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>450</width>
      <height>450</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>450</width>
      <height>138</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="candidateLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayoutCandidate0">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayoutCandidate1">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayoutCandidate2">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionStart_Game"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionboardChanged"/>
    <addaction name="actioncandidateChanged"/>
    <addaction name="actionsetblock"/>
    <addaction name="actionsetcandidates"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuDebug"/>
  </widget>
  <action name="actionStart_Game">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionboardChanged">
   <property name="text">
    <string>boardChanged</string>
   </property>
  </action>
  <action name="actioncandidateChanged">
   <property name="text">
    <string>candidateChanged</string>
   </property>
  </action>
  <action name="actionsetblock">
   <property name="text">
    <string>setblock</string>
   </property>
  </action>
  <action name="actionsetcandidates">
   <property name="text">
    <string>setcandidates</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
